boards/bluepill/include/gpio_params.h

```diff
@@ -34,7 +34,8 @@ static const  saul_gpio_params_t saul_gpio_params[] =
     {
         .name = "LED",
         .pin = LED0_PIN,
-        .mode = GPIO_OUT
+        .mode = GPIO_OUT,
+        .flags = SAUL_GPIO_INVERTED
```

Comment:
could you please add `| SAUL_GPIO_INIT_CLEAR` (see [here](https://github.com/RIOT-OS/RIOT/blob/master/boards/pba-d-01-kw2x/include/gpio_params.h#L44)) as suggested offline yesterday. This makes it explicit that the LED will be off on init when using SAUL.

Reply:
amend changes directly